tests/end2end/experiments/training_plans/sklearn.py

```diff
@@ -0,0 +1,105 @@
+"""
+Sklearn training plans
+"""
+
+from fedbiomed.common.training_plans import FedSGDClassifier, FedPerceptron, FedSGDRegressor
+from fedbiomed.common.data import DataManager
+
+
+from fedbiomed.common.optimizers import Optimizer
+from fedbiomed.common.optimizers.declearn import AdamModule, FedProxRegularizer
+
+
+class PerceptronTraining(FedPerceptron):
+    def training_data(self):
+        NUMBER_COLS = 20
+        dataset = pd.read_csv(self.dataset_path,header=None,delimiter=',')
+        X = dataset.iloc[:,0:NUMBER_COLS].values
+        y = dataset.iloc[:,NUMBER_COLS]
+        return DataManager(dataset=X,target=y.values, shuffle=True)
+
+
+class SGDRegressorTrainingPlan(FedSGDRegressor):
+    def training_data(self):
+        dataset = pd.read_csv(self.dataset_path, delimiter=';', header=0)
```

Comment:
should it be `delimiter=';'` here instead? Like you did for the other training plans

Reply:
In the dataset that is saved in the notebook directory the `delimeter` is `;`

Reply:
Oh ok sorry adni dataset is indeed with `;`, whereas the one generated are with `,`, hence the confusion